src/builtin_bind.cpp

```diff
@@ -35,20 +35,6 @@ struct bind_cmd_opts_t {
     const wchar_t *sets_bind_mode = L"";
 };
 
-// Here follows the definition of all builtin commands. The function names are all of the form
-// builtin_NAME where NAME is the name of the builtin. so the function name for the builtin 'fg' is
-// 'builtin_fg'.
-//
-// A few builtins, including 'while', 'command' and 'builtin' are not defined here as they are
-// handled directly by the parser. (They are not parsed as commands, instead they only alter the
-// parser state)
-//
-// The builtins 'break' and 'continue' are so closely related that they share the same
-// implementation, namely 'builtin_break_continue.
-//
-// Several other builtins, including jobs, ulimit and set are so big that they have been given their
-// own module. These files are all named 'builtin_NAME.cpp', where NAME is the name of the builtin.
-
```

Comment:
Is this supposed to be removed?

Reply:
Yeah, this comment is wrong - it went in there when we split up the builtins.

The commit isn't *really* part of this PR, it's also on master as 056153eddd008233af27904c6ec54320da796882. I forgot to push it before I branched, it'll go away on a rebase.